readthedocs/core/utils/__init__.py

```diff
@@ -74,12 +85,13 @@ def trigger_build(project, version=None, record=True, force=False, basic=False):
             state='triggered',
             success=True,
         )
-        update_docs.delay(pk=project.pk, version_pk=version.pk, record=record,
-                          force=force, basic=basic, build_pk=build.pk)
-    else:
-        build = None
-        update_docs.delay(pk=project.pk, version_pk=version.pk, record=record,
-                          force=force, basic=basic)
+        kwargs['build_pk'] = build.pk
+
+    queue = None
```

Comment:
I just conditionally set the option on `apply_async` instead, in case there is any reason to not set the queue in the future.